src/DataValues/MatrixValue.php

```diff
@@ -0,0 +1,128 @@
+<?php
+namespace DataValues;
+
+use DataValues\DecimalValue;
+/**
+ * Class representing a matrix.
```

Comment:
I meant 2-dimensional array. I have now clarified in the comments that currently it's 2D array of ``DecimalValue``.